app/views/unattended/provisioning_templates/snippet/pxegrub2_chainload.erb

```diff
@@ -67,12 +53,7 @@ echo
   "connectefi #{connectefi_option}" if connectefi_option
 %>
 
-if [ "${lockdown}" == "y" ]; then
-  if [ "${default}" == "local" ]; then
-    set default="grub_config"
-  fi
-
-  menuentry 'Loading GRUB2 config from ESP' --id grub_config {
+menuentry 'Loading GRUB2 from ESP' --id local {
```

Comment:
"Loading system from ESP" would be better here because it first tries to load GRUB2 configuration files, then EFI binaries.